Lib/test/test_bz2.py

```diff
@@ -724,7 +725,8 @@ def testDecompress4G(self, size):
             decompressed = None
 
     def testPickle(self):
-        for proto in range(pickle.HIGHEST_PROTOCOL + 1):
+        # Why?
+        for proto in range(2, pickle.HIGHEST_PROTOCOL + 1):
```

Comment:
@vstinner 
This issue is happening when we port the typeobject into heap type.
Is this ths bug of the implementation?

Reply:
Except this, there was no memory leak on subinterpreter test or module test.